python/semantic_kernel/connectors/memory/azure_cosmosdb/cosmosdb_utils.py

```diff
@@ -29,6 +50,7 @@ def get_mongodb_search_client(connection_string: str):
         raise ServiceInitializationError("Error: missing Azure Cosmos Mongo vCore Connection String")
 
     if cosmos_conn_str:
-        return MongoClient(cosmos_conn_str)
+        app_name = application_name if application_name is not None else "PYTHON_SEMANTIC_KERNEL"
```

Comment:
Same comment here, for application name we have dedicated constant which we use across different connectors:
https://github.com/microsoft/semantic-kernel/blob/41651ff2bef1d97f9ac5ffa37a68123418814588/python/semantic_kernel/connectors/telemetry.py#L13